gems/pending/spec/appliance_console/logfile_configuration_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "appliance_console/logfile_configuration"
+
+describe ApplianceConsole::LogfileConfiguration do
+  before do
+    @spec_name = File.basename(__FILE__).split(".rb").first.freeze
+    allow(LinuxAdmin::Service).to receive(:new).and_return(double(@spec_name, :running? => false))
+    allow(subject).to receive(:clear_screen)
+    allow(subject).to receive(:say)
+  end
+
+  describe "#ask_questions" do
+    context "when user confirms a new disk" do
+      before do
+        expect(subject).to receive(:agree).and_return(true).twice
+        expect(subject).to receive(:ask_for_disk).and_return(double(:@spec_name, :size => "22", :path => "fake disk"))
+      end
+
+      it "returns true" do
+        expect(subject.ask_questions).to be true
+      end
+    end
+
+    context "when user does not confirm a new disk" do
+      before do
+        expect(subject).to receive(:agree).and_return(false)
+        expect(subject).to_not receive(:ask_for_disk)
+      end
+
+      it "returns false" do
+        expect(subject.ask_questions).to be false
+      end
+    end
+
+    context "when user does not confirm selection" do
+      before do
+        expect(subject).to receive(:agree).with("Configure a new logfile disk volume? (Y/N):").and_return(true)
+        expect(subject).to receive(:agree).with(/Continue with disk:/).and_return(false)
+        expect(subject).to receive(:ask_for_disk).and_return(double(:@spec_name, :size => "22", :path => "fake disk"))
+      end
+
+      it "returns false" do
+        expect(subject.ask_questions).to be false
+      end
+    end
+  end
+
+  describe "#activate" do
+    before do
```

Comment:
@bdunne and @carbonin Agreed. I initially coded the

`describe do, context do, it do` framework not knowing how may tests there would end up being. I will refactoring to simplify.